packages/css/gulpfile.js

```diff
@@ -103,36 +109,35 @@ async function createSvgSpritesheet() {
             const stylesheet = stylesheets.find(s => s.id === id);
             const iconIds = [
               id,
-              ...(stylesheet ? stylesheet.variants : []).map(v => `${id}-${v}`)
+              ...(stylesheet ? stylesheet.variants.filter(v => v.variant !== 'default') : []).map(v => `${id}-${v.variant}`)
             ];
 
             symbol
               .before(`<!-- START: ${iconIds.join(', ')} -->`)
               .after(`<!-- END: ${iconIds.join(', ')} -->`);
 
-            if (stylesheet) {
-              const style = $('<style>')
-                .attr('type', 'text/css')
-                .html(`\n${indent(stylesheet.style, 4)}\n  `); // last two spaces are indent fix
-
-              symbol.before(style);
-            }
-
             iconIds.forEach((iconId, index) => {
               const x = (position + index) * (canvas + gutter) + gutter / 2;
+              const svg = symbol.clone().removeAttr('id');
+              $(svg).each((_index, element) => element.tagName = 'svg');
+
+              if (stylesheet) {
+                variantColor = stylesheet.variants.find(v => iconId.includes(v.variant))?.color || stylesheet.variants.find(v => v.variant === 'default')?.color;
```

Comment:
Ik begrijp niet wat dit doet.

Ik heb 't idee dat we hier een verkeerde aanname doen: `default` is geen fallback waarde maar het "gewone" basis icoon. Het is niet de bedoeling dat ie voor 'n variant geen kleur kan vinden, dat ie dan maar `default` gebruikt.

Misschien moeten we strenger zijn: Als je een variant aanmaakt (`icon:variant {}`) en geen `color`, gooi dan maar 'n error (`throw new Error('Variant heeft geen color');`) oid.

Maandag samen kijken?

Reply:
Voorstel is om iconIds iteratie te vervangen met stylesheet.variants.